docs/topics/wasm/wasm-debugging.md

```diff
@@ -133,41 +133,42 @@ kotlin {
 >
 {style="note"}
 
-#### Enhance your debugging with custom formatters
+#### Use custom formatters
 
-In addition to the default debugging configuration, you can set up custom formatters to display and locate variable values in a more user-friendly and comprehensible manner.
+Custom formatters help display and locate variable values in a more user-friendly and comprehensible manner when debugging Kotlin/Wasm code.
 
-![Kotlin/Wasm improved debugger](wasm-debugger-improved.png){width=600}
+Custom formatters are enabled by default in development builds, so you don't need additional Gradle configurations. 
 
-This implementation is supported across major browsers like Firefox and Chromium-based as
+This feature is supported across major browsers like Firefox and Chromium-based as
```

Comment:
This could be clearer, I think: "...is supported in Firefox and Chromium-based browsers." Because further we don't talk about anything else, and from a cursory search I'm not sure other major browsers, like Safari, support this at all.